app/src/main/res/values-fr/strings.xml

```diff
@@ -9,8 +9,8 @@
         <item name="valor">"Bravoure"</item>
         <item name="instinct">"Intuition"</item>
     </string-array>
-    <string name="place1">"Dans l\'ensemble, ton Pokémon&#8230;"</string>
-    <string name="place2">"Ses stats&#8230;"</string>
+    <string name="place1">Dans l\'ensemble, ton Pokémon&#8230;</string>
+    <string name="place2">Ses stats&#8230;</string>
```

Comment:
Based on my understanding of the syntax, this looks like actually a bugfix (the current code will show a `\'` to the user). @Remiscan @toinux or does this look right?